lib/epicenter_web/presenters/contact_investigation_presenter.ex

```diff
@@ -8,6 +8,7 @@ defmodule EpicenterWeb.Presenters.ContactInvestigationPresenter do
   alias EpicenterWeb.Format
   alias EpicenterWeb.Presenters.PeoplePresenter
   alias EpicenterWeb.Router.Helpers, as: Routes
+  use Phoenix.VerifiedRoutes, endpoint: EpicenterWeb.Endpoint, router: EpicenterWeb.Router
```

Comment:
If there are many presenters, then we may want create and use a `EpiCenterWeb.presenter/0` in `lib/epi_center_web.ex`

Reply:
added